src/main.rs

```diff
@@ -251,6 +258,8 @@ fn process<P, I>(old_args: I, dep_path: P, sysroot: &str) -> Result<(), i32>
     args.push(String::from("--sysroot"));
     args.push(sysroot.to_owned());
     args.push("-Zno-trans".to_owned());
+    args.push("--cfg".to_owned());
+    args.push(r#"feature="cargo-clippy""#.to_owned());
```

Comment:
Nah, cargo features are checked, but rustc features are not. Currently `cargo clippy --features=nope` gives
> error: Package `clippy v0.0.96 (file:///…)` does not have these features: `nope`

but `cargo clippy -- --cfg 'feature="nope"'` works fine.